index.html

```diff
@@ -11,12 +11,20 @@
       href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
     />
     <link rel="stylesheet" href="/css/style.css" />
+    <!-- normalize.css cdn 이용 -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <title>판다마켓</title>
   </head>
   <body>
     <!-- header -->
-    <header id="header">
-      <a href="#">
+    <header>
+      <a href="/">
```

Comment:
💊 제안
id 를 이용하지 않고 css 스타일링을 해주신 것은 좋지만, 태그 선택자는 해당 태그에 모든 페이지에 걸쳐 적용되야 하는 스타일링 외에는 사용을 추천드리지 않습니다~
가능하면 class를 사용하시는 것을 추천드릴께요~

```suggestion
    <header class="header">
      <a href="/">
```